Source/Core/Core/ConfigManager.cpp

```diff
@@ -87,6 +87,17 @@ void SConfig::SaveSettings()
   m_SYSCONF->Save();
 }
 
+static void CreateDumpPath(const std::string& path)
```

Comment:
Prefer putting stuff in an anonymous namespace instead of "static". It's cleaner. There is also a case to be made for this to be a static member function of SConfig too. Easier way works for me.